sourcemash/api/feeds.py

```diff
@@ -104,8 +104,8 @@ def post(self):
                 q = Queue('scrape', connection=REDIS_CONNECTION)
```

Comment:
Better idea: Make a dead_connection fixture in app fixture that sets environment var REDIS_URL to localhost with some incorrect port.

Then write a test with dead_connection.

See: https://docs.python.org/2/library/os.html#os.putenv